vhdbuilder/packer/install-dependencies.sh

```diff
@@ -318,11 +320,16 @@ KUBE_PROXY_IMAGE_VERSIONS="
 1.19.7-hotfix.20210310.2
 1.19.9-hotfix.20210322
 1.19.9-hotfix.20210322.1
+1.19.9-hotfix.20210505
+1.19.10-hotfix.20210504
 1.20.2
 1.20.2-hotfix.20210310
 1.20.2-hotfix.20210310.2
 1.20.5-hotfix.20210322
 1.20.5-hotfix.20210322.1
+1.20.5-hotfix.20210505
+1.20.6-hotfix.20210504
+1.21.0
```

Comment:
Will we support 1.21.0? If so, I will add this in AKS Windows VHD

Reply:
yes we will